### PR TITLE
Drop PHP 7.3 support

### PR DESCRIPTION
**Pull request by Slamdunk**
Signed-off-by: Filippo Tessarotto <zoeslam@gmail.com>

### CODE REVIEW

**Comment by glensc** [comment]
please no.

File: composer.json [hunk 1]
```diff
@@ -8,32 +8,34 @@
...
-        "laminas/laminas-validator": "^2.15",
-        "symfony/polyfill-mbstring": "^1.12.0",
-        "webmozart/assert": "^1.10",
-        "symfony/polyfill-intl-idn": "^1.24.0"
+        "laminas/laminas-stdlib": "^3.7.0",
+        "laminas/laminas-validator": "^2.15.1",
+        "symfony/polyfill-intl-idn": "^1.24.0",
+        "symfony/polyfill-mbstring": "^1.24.0",
+        "webmozart/assert": "^1.10.0"
     },
```
Comment (glensc) [inline composer.json:20]: the diff is massively different than the commit message says.

have you thought of creating meaningful and atomic commits?

1. Make [atomic commits]
2. Write [good commit messages]


[good commit messages]: http://chris.beams.io/posts/git-commit/
[atomic commits]: http://www.freshconsulting.com/atomic-commits/

Comment (Ocramius) [reply composer.json:20]: The diff seems to just drop dependencies that no longer operate well with latest PHP versions?

**Review by Ocramius — APPROVED** [review]

**Comment by Slamdunk** [comment]
@glensc first things first, I want to thank you for your commitment into monitor and improving this library: it didn't go unnoticed in these years.

> please no.

Without an explanation, this doesn't change much my mind about moving forward the library

> the diff is massively different than the commit message says.
> have you thought of creating meaningful and atomic commits?

You are right, I'm guilty of squashed commits, sorry

@Ocramius the bump in `laminas-coding-standards` brings with it `declare(strict_types=1);` everywhere. It seems to me a BC break, so I'm going to suppress that CS fixer for the current major version: am I right?

**Comment by Ocramius** [comment]
Depends on how big the psalm baseline is: strict types mostly affect internal calls and inheritance, not regular client-to-library interactions

**Pull request closed without merging**